[2025-01-17T10:58:58.403+0000] {processor.py:186} INFO - Started process (PID=65) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T10:58:58.409+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T10:58:58.422+0000] {logging_mixin.py:190} INFO - [2025-01-17T10:58:58.421+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T10:58:58.475+0000] {logging_mixin.py:190} INFO - [2025-01-17T10:58:58.468+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T10:58:58.477+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T10:58:58.537+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.168 seconds
[2025-01-17T10:59:29.148+0000] {processor.py:186} INFO - Started process (PID=137) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T10:59:29.152+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T10:59:29.159+0000] {logging_mixin.py:190} INFO - [2025-01-17T10:59:29.158+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T10:59:29.202+0000] {logging_mixin.py:190} INFO - [2025-01-17T10:59:29.201+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T10:59:29.203+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T10:59:29.237+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.100 seconds
[2025-01-17T11:00:00.110+0000] {processor.py:186} INFO - Started process (PID=204) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:00:00.111+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:00:00.113+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:00:00.113+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:00:00.130+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:00:00.129+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:00:00.131+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:00:00.146+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.040 seconds
[2025-01-17T11:00:30.971+0000] {processor.py:186} INFO - Started process (PID=271) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:00:30.973+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:00:30.981+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:00:30.980+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:00:31.020+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:00:31.018+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:00:31.021+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:00:31.058+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.100 seconds
[2025-01-17T11:01:01.718+0000] {processor.py:186} INFO - Started process (PID=338) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:01:01.719+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:01:01.721+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:01:01.720+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:01:01.740+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:01:01.739+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:01:01.740+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:01:01.756+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.044 seconds
[2025-01-17T11:01:31.901+0000] {processor.py:186} INFO - Started process (PID=405) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:01:31.903+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:01:31.906+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:01:31.905+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:01:31.925+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:01:31.924+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:01:31.926+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:01:31.945+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.049 seconds
[2025-01-17T11:02:01.983+0000] {processor.py:186} INFO - Started process (PID=473) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:02:01.984+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:02:01.986+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:02:01.986+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:02:02.006+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:02:02.005+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:02:02.006+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:02:02.034+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.055 seconds
[2025-01-17T11:02:32.587+0000] {processor.py:186} INFO - Started process (PID=541) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:02:32.588+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:02:32.591+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:02:32.590+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:02:32.616+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:02:32.614+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:02:32.617+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:02:32.635+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.054 seconds
[2025-01-17T11:03:03.166+0000] {processor.py:186} INFO - Started process (PID=608) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:03:03.167+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:03:03.169+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:03:03.169+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:03:03.197+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:03:03.196+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:03:03.197+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:03:03.213+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.052 seconds
[2025-01-17T11:03:33.321+0000] {processor.py:186} INFO - Started process (PID=675) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:03:33.323+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:03:33.326+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:03:33.326+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:03:33.363+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:03:33.362+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:03:33.364+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:03:33.385+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.070 seconds
[2025-01-17T11:04:03.564+0000] {processor.py:186} INFO - Started process (PID=742) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:04:03.565+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:04:03.566+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:04:03.566+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:04:03.582+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:04:03.581+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:04:03.583+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:04:03.609+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.051 seconds
[2025-01-17T11:04:33.911+0000] {processor.py:186} INFO - Started process (PID=809) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:04:33.912+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:04:33.915+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:04:33.914+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:04:33.933+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:04:33.932+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:04:33.933+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:04:33.961+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.055 seconds
[2025-01-17T11:05:04.050+0000] {processor.py:186} INFO - Started process (PID=876) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:05:04.052+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:05:04.054+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:05:04.054+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:05:04.073+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:05:04.072+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:05:04.074+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:05:04.090+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.044 seconds
[2025-01-17T11:05:34.883+0000] {processor.py:186} INFO - Started process (PID=943) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:05:34.884+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:05:34.886+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:05:34.886+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:05:34.903+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:05:34.902+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:05:34.904+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:05:34.931+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.052 seconds
[2025-01-17T11:06:05.186+0000] {processor.py:186} INFO - Started process (PID=1010) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:06:05.187+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:06:05.188+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:06:05.188+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:06:05.206+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:06:05.205+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:06:05.206+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:06:05.233+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.053 seconds
[2025-01-17T11:06:35.750+0000] {processor.py:186} INFO - Started process (PID=1077) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:06:35.752+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:06:35.755+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:06:35.754+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:06:35.771+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:06:35.770+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:06:35.772+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:06:35.788+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.043 seconds
[2025-01-17T11:07:05.932+0000] {processor.py:186} INFO - Started process (PID=1144) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:07:05.933+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:07:05.936+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:07:05.935+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:07:05.955+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:07:05.954+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:07:05.956+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:07:05.975+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.048 seconds
[2025-01-17T11:07:36.347+0000] {processor.py:186} INFO - Started process (PID=1211) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:07:36.349+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:07:36.351+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:07:36.351+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:07:36.370+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:07:36.369+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:07:36.371+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:07:36.388+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.046 seconds
[2025-01-17T11:08:06.591+0000] {processor.py:186} INFO - Started process (PID=1278) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:08:06.592+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:08:06.594+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:08:06.594+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:08:06.610+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:08:06.610+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:08:06.611+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:08:06.636+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.050 seconds
[2025-01-17T11:08:37.366+0000] {processor.py:186} INFO - Started process (PID=1345) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:08:37.367+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:08:37.369+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:08:37.369+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:08:37.390+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:08:37.389+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:08:37.391+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:08:37.420+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.058 seconds
[2025-01-17T11:09:07.657+0000] {processor.py:186} INFO - Started process (PID=1412) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:09:07.658+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:09:07.660+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:09:07.660+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:09:07.678+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:09:07.677+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:09:07.679+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:09:07.696+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.044 seconds
[2025-01-17T11:09:38.479+0000] {processor.py:186} INFO - Started process (PID=1479) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:09:38.481+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:09:38.482+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:09:38.482+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:09:38.497+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:09:38.496+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:09:38.498+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:09:38.525+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.051 seconds
[2025-01-17T11:10:09.134+0000] {processor.py:186} INFO - Started process (PID=1545) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:10:09.136+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:10:09.138+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:10:09.137+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:10:09.160+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:10:09.159+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:10:09.161+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:10:09.178+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.049 seconds
[2025-01-17T11:10:39.689+0000] {processor.py:186} INFO - Started process (PID=1611) to work on /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:10:39.690+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Module/CapstoneETL.py for tasks to queue
[2025-01-17T11:10:39.692+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:10:39.692+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:10:39.710+0000] {logging_mixin.py:190} INFO - [2025-01-17T11:10:39.710+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/Module/CapstoneETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Module/CapstoneETL.py", line 132
    %pip install psycopg2
    ^
SyntaxError: invalid syntax
[2025-01-17T11:10:39.711+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Module/CapstoneETL.py
[2025-01-17T11:10:39.738+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Module/CapstoneETL.py took 0.053 seconds
